### ISSUE TITLE
CoveragePlot does not show genes

### ISSUE THREAD

**Issue by hchintalapudi** [timoast/signac#381]
Hi,
I'm using the WNN analysis vignette (https://satijalab.org/seurat/v4.0/weighted_nearest_neighbor_analysis.html) to analyze our 10x multimodal (ATAC+RNA) PBMC data and did the annotation using the multimodal reference mapping analysis (https://satijalab.org/seurat/v4.0/reference_mapping.html)

I was trying to see the coverage plots to examine peaks with associated gene exp plots and the command I used was:
CoveragePlot(pbmc, region = 'chr9-130171653-130173919', features = "NCS1",assay = 'ATAC', expression.assay = "SCT", peaks = F, extend.upstream = 4000,extend.downstream = 2000)
which produces:
[tmp_for_GitHub.pdf](https://github.com/timoast/signac/files/5767150/tmp_for_GitHub.pdf)
Error:
Warning messages:
1: In .Seqinfo.mergexy(x, y) :
  The 2 combined objects have no sequence levels in common. (Use
  suppressWarnings() to suppress this warning.)
2: Removed 2 rows containing missing values (position_stack).

The steps for running the analysis were exactly same as in WNN vignette up until the annotation obviously.
Any tips appreciated.
Thanks!

**Comment by timoast** [comment]
Can you provide more information? What code are you running?

**Comment by timoast** [comment]
Did you add gene annotations to the assay? Can you post the full code and output of `sessionInfo()`?

**Comment by hchintalapudi** [comment]
Full code:
# the 10x hdf5 file contains both data types.
inputdata.10x <- Read10X_h5("/Users/hchintalapudi/Desktop/ATAC-Seq/Multiome_HumanPBMCs/CBNMC_1/outs/filtered_feature_bc_matrix.h5")
# extract RNA and ATAC data
rna_counts <- inputdata.10x$`Gene Expression`
atac_counts <- inputdata.10x$Peaks

# Create Seurat object
pbmc <- CreateSeuratObject(counts = rna_counts)
pbmc[["percent.mt"]] <- PercentageFeatureSet(pbmc, pattern = "^MT")

# Now add in the ATAC-seq data
# we'll only use peaks in standard chromosomes
grange.counts <- StringToGRanges(rownames(atac_counts), sep = c(":", "-"))
grange.use <- seqnames(grange.counts) %in% standardChromosomes(grange.counts)
atac_counts <- atac_counts[as.vector(grange.use), ]
annotations <- GetGRangesFromEnsDb(ensdb = EnsDb.Hsapiens.v86)
seqlevelsStyle(annotations) <- 'UCSC'
genome(annotations) <- "hg38"
frag.file <-"/Users/hchintalapudi/Desktop/ATAC-Seq/Multiome_HumanPBMCs/CBNMC_1/outs/atac_fragments.tsv.gz"
chrom_assay <- CreateChromatinAssay(
   counts = atac_counts,
   sep = c(":", "-"),
   genome = 'hg38',
   fragments = frag.file,
   min.cells = 10,
   annotation = annotations
 )
pbmc[["ATAC"]] <- chrom_assay

##Basic QC:
VlnPlot(pbmc, features = c("nCount_ATAC", "nCount_RNA","percent.mt"), ncol = 3,
  log = TRUE, pt.size = 0) + NoLegend()

pbmc
pbmc <- subset(
  x = pbmc,
  subset = nCount_ATAC < 8e4 &
    nCount_ATAC > 5e3 &
    nCount_RNA < 10000 &
    nCount_RNA > 200 &
    percent.mt < 20
)

# RNA analysis
DefaultAssay(pbmc) <- "RNA"
pbmc <- SCTransform(pbmc, verbose = FALSE) %>% RunPCA() %>% RunUMAP(dims = 1:50, reduction.name = 'umap.rna', reduction.key = 'rnaUMAP_')

# ATAC analysis
# We exclude the first dimension as this is typically correlated with sequencing depth
DefaultAssay(pbmc) <- "ATAC"
pbmc <- RunTFIDF(pbmc)
pbmc <- FindTopFeatures(pbmc, min.cutoff = 'q0')
pbmc <- RunSVD(pbmc)
pbmc <- RunUMAP(pbmc, reduction = 'lsi', dims = 2:50, reduction.name = "umap.atac", reduction.key = "atacUMAP_")


pbmc <- FindMultiModalNeighbors(pbmc, reduction.list = list("pca", "lsi"), dims.list = list(1:50, 2:50))
pbmc <- RunUMAP(pbmc, nn.name = "weighted.nn", reduction.name = "wnn.umap", reduction.key = "wnnUMAP_")
pbmc <- FindClusters(pbmc, graph.name = "wsnn", algorithm = 3, verbose = FALSE)

reference <- LoadH5Seurat("/Users/hchintalapudi/Desktop/ATAC-Seq/Multiome_HumanPBMCs/pbmc_multimodal.h5seurat")


DimPlot(object = reference, reduction = "wnn.umap", group.by = "celltype.l2", label = TRUE, label.size = 3, repel = TRUE) +  ggtitle("Reference PBMCs") + NoLegend()

DefaultAssay(pbmc) <- "RNA"
pbmc<- SCTransform(pbmc, verbose = FALSE)
anchors <- FindTransferAnchors(
  reference = reference,
  query = pbmc,
  normalization.method = "SCT",
  reference.reduction = "spca",
  dims = 1:50
)

pbmc <- MapQuery(
  anchorset = anchors,
  query = pbmc,
  reference = reference,
  refdata = list(
    celltype.l1 = "celltype.l1",
    celltype.l2 = "celltype.l2",
    predicted_ADT = "ADT"),
  reference.reduction = "spca", 
  reduction.model = "wnn.umap"
)

> sessionInfo()
R version 4.0.3 (2020-10-10)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Catalina 10.15.2

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] EnsDb.Hsapiens.v86_2.99.0 ensembldb_2.12.1          AnnotationFilter_1.12.0   GenomicFeatures_1.40.1   
 [5] AnnotationDbi_1.50.3      Biobase_2.48.0            scCATCH_2.1               patchwork_1.1.1          
 [9] SeuratDisk_0.0.0.9013     forcats_0.5.0             stringr_1.4.0             dplyr_1.0.2              
[13] purrr_0.3.4               readr_1.4.0               tidyr_1.1.2               tibble_3.0.4             
[17] tidyverse_1.3.0           survminer_0.4.8.999       ggpubr_0.4.0              survival_3.2-7           
[21] writexl_1.3.1             readxl_1.3.1              ggplot2_3.3.2             Signac_1.1.0             
[25] Seurat_3.9.9.9024         GenomicRanges_1.40.0      GenomeInfoDb_1.24.2       IRanges_2.22.2           
[29] S4Vectors_0.26.1          BiocGenerics_0.34.0      

loaded via a namespace (and not attached):
  [1] rappdirs_0.3.1              SnowballC_0.7.0             rtracklayer_1.48.0          scattermore_0.7            
  [5] GGally_2.0.0                bit64_4.0.5                 knitr_1.30                  irlba_2.3.3                
  [9] DelayedArray_0.14.1         data.table_1.13.4           rpart_4.1-15                RCurl_1.98-1.2             
 [13] generics_0.1.0              callr_3.5.1                 cowplot_1.1.0               usethis_1.6.3              
 [17] RSQLite_2.2.1               RANN_2.6.1                  future_1.21.0               bit_4.0.4                  
 [21] lubridate_1.7.9.2           spatstat.data_1.7-0         xml2_1.3.2                  httpuv_1.5.4               
 [25] SummarizedExperiment_1.18.2 assertthat_0.2.1            xfun_0.19                   hms_0.5.3                  
 [29] evaluate_0.14               promises_1.1.1              fansi_0.4.1                 progress_1.2.2             
 [33] dbplyr_2.0.0                km.ci_0.5-2                 igraph_1.2.6                DBI_1.1.0                  
 [37] htmlwidgets_1.5.3           reshape_0.8.8               ellipsis_0.3.1              RSpectra_0.16-0            
 [41] backports_1.2.1             biomaRt_2.44.4              deldir_0.2-3                vctrs_0.3.6                
 [45] remotes_2.2.0               ROCR_1.0-11                 abind_1.4-5                 withr_2.3.0                
 [49] ggforce_0.3.2               BSgenome_1.56.0             checkmate_2.0.0             sctransform_0.3.2          
 [53] GenomicAlignments_1.24.0    prettyunits_1.1.1           goftest_1.2-2               cluster_2.1.0              
 [57] lazyeval_0.2.2              crayon_1.3.4                hdf5r_1.3.3                 labeling_0.4.2             
 [61] pkgconfig_2.0.3             tweenr_1.0.1                pkgload_1.1.0               nlme_3.1-150               
 [65] ProtGenerics_1.20.0         devtools_2.3.2              nnet_7.3-14                 rlang_0.4.9                
 [69] globals_0.14.0              lifecycle_0.2.0             miniUI_0.1.1.1              BiocFileCache_1.12.1       
 [73] modelr_0.1.8                rsvd_1.0.3                  dichromat_2.0-0             cellranger_1.1.0           
 [77] rprojroot_2.0.2             polyclip_1.10-0             matrixStats_0.57.0          lmtest_0.9-38              
 [81] graph_1.66.0                Matrix_1.2-18               ggseqlogo_0.1               KMsurv_0.1-5               
 [85] carData_3.0-4               zoo_1.8-8                   reprex_0.3.0                base64enc_0.1-3            
 [89] ggridges_0.5.2              processx_3.4.5              png_0.1-7                   viridisLite_0.3.0          
 [93] bitops_1.0-6                KernSmooth_2.23-18          Biostrings_2.56.0           blob_1.2.1                 
 [97] parallelly_1.22.0           jpeg_0.1-8.1                rstatix_0.6.0               ggsignif_0.6.0             
[101] scales_1.1.1                memoise_1.1.0               magrittr_2.0.1              plyr_1.8.6                 
[105] ica_1.0-2                   zlibbioc_1.34.0             compiler_4.0.3              RColorBrewer_1.1-2         
[109] fitdistrplus_1.1-3          Rsamtools_2.4.0             cli_2.2.0                   XVector_0.28.0             
[113] listenv_0.8.0               pbapply_1.4-3               ps_1.5.0                    htmlTable_2.1.0            
[117] Formula_1.2-4               MASS_7.3-53                 mgcv_1.8-33                 tidyselect_1.1.0           
[121] stringi_1.5.3               yaml_2.2.1                  askpass_1.1                 latticeExtra_0.6-29        
[125] ggrepel_0.9.0               survMisc_0.5.5              grid_4.0.3                  VariantAnnotation_1.34.0   
[129] fastmatch_1.1-0             tools_4.0.3                 future.apply_1.6.0          rio_0.5.16                 
[133] rstudioapi_0.13             foreign_0.8-80              lsa_0.73.2                  gridExtra_2.3              
[137] farver_2.0.3                Rtsne_0.15                  digest_0.6.27               BiocManager_1.30.10        
[141] shiny_1.5.0                 Rcpp_1.0.5                  car_3.0-10                  broom_0.7.3                
[145] later_1.1.0.1               RcppAnnoy_0.0.18            OrganismDbi_1.30.0          httr_1.4.2                 
[149] ggbio_1.36.0                biovizBase_1.36.0           colorspace_2.0-0            rvest_0.3.6                
[153] fs_1.5.0                    XML_3.99-0.5                tensor_1.5                  reticulate_1.18            
[157] splines_4.0.3               uwot_0.1.10                 RBGL_1.64.0                 RcppRoll_0.3.0             
[161] spatstat.utils_1.17-0       sessioninfo_1.1.1           plotly_4.9.2.2              xtable_1.8-4               
[165] jsonlite_1.7.2              spatstat_1.64-1             testthat_3.0.1              R6_2.5.0                   
[169] Hmisc_4.4-2                 pillar_1.4.7                htmltools_0.5.0             mime_0.9                   
[173] glue_1.4.2                  fastmap_1.0.1               BiocParallel_1.22.0         codetools_0.2-18           
[177] pkgbuild_1.2.0              lattice_0.20-41             curl_4.3                    leiden_0.3.6               
[181] zip_2.1.1                   openxlsx_4.2.3              openssl_1.4.3               limma_3.44.3               
[185] rmarkdown_2.5               desc_1.2.0                  munsell_0.5.0               GenomeInfoDbData_1.2.3     
[189] haven_2.3.1                 reshape2_1.4.4              gtable_0.3.0

**Comment by timoast** [comment]
Are you sure the data were mapped to hg38? At what point do you get the error you mentioned above? Can you show the traceback from the error?

**Comment by hchintalapudi** [comment]
Yes, I used EnsDb.Hsapiens.v86 as in the vignette.
annotations <- GetGRangesFromEnsDb(ensdb = EnsDb.Hsapiens.v86)
genome(annotations) <- "hg38"
> head(annotations)
GRanges object with 6 ranges and 5 metadata columns:
                  seqnames        ranges strand |           tx_id   gene_name         gene_id   gene_biotype     type
                     <Rle>     <IRanges>  <Rle> |     <character> <character>     <character>    <character> <factor>
  ENSE00001489430        X 276322-276394      + | ENST00000399012      PLCXD1 ENSG00000182378 protein_coding     exon
  ENSE00001536003        X 276324-276394      + | ENST00000484611      PLCXD1 ENSG00000182378 protein_coding     exon
  ENSE00002160563        X 276353-276394      + | ENST00000430923      PLCXD1 ENSG00000182378 protein_coding     exon
  ENSE00001750899        X 281055-281121      + | ENST00000445062      PLCXD1 ENSG00000182378 protein_coding     exon
  ENSE00001489388        X 281192-281684      + | ENST00000381657      PLCXD1 ENSG00000182378 protein_coding     exon
  ENSE00001719251        X 281194-281256      + | ENST00000429181      PLCXD1 ENSG00000182378 protein_coding     exon
  -------
  seqinfo: 25 sequences from hg38 genome

I deleted my Seurat obj and reran the code again till clustering and did not do the mapping to the reference dataset. 
And tried the CoveragePlot() func only to get the same plot with empty genes track and the same error:
Warning message:
In .Seqinfo.mergexy(x, y) :
  The 2 combined objects have no sequence levels in common. (Use
  suppressWarnings() to suppress this warning.)
